src/data/people.js

```diff
@@ -64,7 +64,7 @@ export default {
     lastName: 'Seeger',
     nickname: 'Brandon',
     avatar: './images/brandon.png',
-    orgID: 'org1',
+    orgID: 'cisco',
```

Comment:
Because of changes from https://github.com/webex/components/pull/182 make sure to reconcile the [data where `user5` is part of `org2`](https://github.com/webex/components/blob/master/src/data/people.js#L57)